<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysWindow</class>
 <widget class="QMainWindow" name="SysWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统管理员</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recourse.qrc">
    <normaloff>:/icon1.png</normaloff>:/icon1.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(26,148,188);
border:none</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="BottomGB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>800</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:none</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="LogGB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(20, 196, 188);
border-radius:30px;
background-color:rgb(255,255,255);

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="SysStrL">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>3</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 15pt &quot;Microsoft YaHei UI&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string>人员管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="SysInfoRb">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>450</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">back-ground-color:rgb(255,255,255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string>个人信息</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MgrRb">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>450</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">back-ground-color:rgb(255,255,255);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string>人员管理</string>
     </property>
    </widget>
    <widget class="QTableView" name="dataLV">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>139</y>
       <width>600</width>
       <height>311</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView{
	border:2px solid rgb(20,196,188);
	border-radius:0px;
	selection-color: white;                             /*选中区域的文字颜色*/
    selection-background-color:rgb(20,196,188);
}
QHeaderView::section {
    color: black;
    background-color: rgb(255,255,255);
    border: 0px solid rgb(20,196,188);
    border-radius:0px;
    border-color: rgb(20,196,188);
} 
QHeaderView{
	border:none;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>27</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="NameL">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>220</y>
       <width>390</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 10pt &quot;Microsoft YaHei UI&quot;;
border:none;
border-radius:20px</string>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="TeleL">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>350</y>
       <width>390</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 10pt &quot;Microsoft YaHei UI&quot;;
border:none;
border-radius:20px</string>
     </property>
     <property name="text">
      <string>联系方式：</string>
     </property>
    </widget>
    <widget class="QLabel" name="WaringL">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>470</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 9pt &quot;Microsoft YaHei UI&quot;;
color:rgb(204,22,58);
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="IdL">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>90</y>
       <width>390</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 10pt &quot;Microsoft YaHei UI&quot;;
border:none;
border-radius:20px</string>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InfoT">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>481</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
border:2px solid rgb(20,196,188);
font: 290 10pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="AddB">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearhB">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>90</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ModfiyB">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>90</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteB">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>90</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FinshB">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(20,196,188);
	color: rgb(255,255,255);
	font: 25 14pt &quot;Microsoft YaHei UI&quot;;
	border: none;
	border-radius:20px;
}

QPushButton:hover{
		background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}


</string>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
    <zorder>dataLV</zorder>
    <zorder>SysStrL</zorder>
    <zorder>SysInfoRb</zorder>
    <zorder>MgrRb</zorder>
    <zorder>NameL</zorder>
    <zorder>TeleL</zorder>
    <zorder>WaringL</zorder>
    <zorder>IdL</zorder>
    <zorder>InfoT</zorder>
    <zorder>AddB</zorder>
    <zorder>SearhB</zorder>
    <zorder>ModfiyB</zorder>
    <zorder>DeleteB</zorder>
    <zorder>FinshB</zorder>
   </widget>
   <widget class="QPushButton" name="MinB">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
		background-color:rgb(21, 131, 161);
}
QPushButton:pressed{
	background-color:rgb(17, 111, 134);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recourse.qrc">
      <normaloff>:/icon3.png</normaloff>:/icon3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>19</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseB">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
		background-color: rgb(255, 0, 0);
}
QPushButton:pressed{
	background-color: rgb(162, 0, 3);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recourse.qrc">
      <normaloff>:/icon2.png</normaloff>:/icon2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recourse.qrc"/>
 </resources>
 <connections/>
</ui>
